cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,18 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden. in_entrance_exam is inherited metadata, all children will have it.
```

Comment:
I am still confused why lines 821-824 are necessary. I understand you want to check specifically for xblock.category == 'sequential', and I agree that should stay. But if the condition "xblock.category == 'sequential' and getattr(xblock, "in_entrance_exam", False)" is True, won't it always be the case that if getattr(parent_xblock, "is_entrance_exam", None) is also true? To state it another way, is there any case where you can have a sequential with in_entrance_exam == True, but its parent has in_entrance_exam == False?